ep/build-velox/src/setup-centos7.sh

```diff
@@ -48,14 +48,29 @@ function yum_install {
 
 function wget_and_untar {
```

Comment:
@yaooqinn, thanks for work!

I note upstream velox's wget_and_untar has a check for dir to let user decide whether to re-download the tar. Does it meet our needs?

Actually, by removing this `wget_and_untar` function, Velox's `wget_and_untar` (placed in `setup-helper-functions.sh`) will be used in this script. I am wondering whether this improvement can be put in upstream velox. Then, both `setup-centos7.sh` and `setup-centos8.sh` (maintained in Gluten) can use that, as well as `setup-centos9.sh` and `setup-ubuntu.sh` (maintained in Velox).

Reply:
Hi, @PHILO-HE. Thanks for the suggestion.

I had considered using velox's wget_and_untar, but I didn't apply it here. 

- Some of the dependency names might conflict as these names are in the form of `vx.y.z`. 
- velox's wget_and_untar does not check whether the tar ball is valid or not.

> I am wondering whether this improvement can be put in upstream velox.

This is a good suggestion, I will try to send a PR to the upstream. But how about we focus the dependency management here first

Reply:
On second thought, I agree w/ you about removing the wget_and_untar and use the upstream one